lib/commands/dialog/index.js

```diff
@@ -125,7 +125,7 @@ class DialogCommand extends AbstractCommand {
                 Messenger.getInstance().info(`Defaulting locale to the first value from the skill manifest: ${firstLocaleFromManifest}`);
             }
             locale = locale || process.env.ASK_DEFAULT_DEVICE_LOCALE || firstLocaleFromManifest;
-            const simulationType = !stringUtils.isNonBlankString(cmd.simulationType) ? 'DEFAULT' : cmd.simulationType;
+            const simulationType = stringUtils.isNonBlankString(cmd.simulationType) ? cmd.simulationType : undefined;
```

Comment:
Why is `DEFAULT` no longer the default value?

Reply:
The `simulation.type` in simulate-skill request should only be set when it's to simulate a NFI enabled skill. We should not set this field at all rather then letting developers set this field explicitly. 

The meaning of the `DEFAULT` here is ambiguous, which we have requested the service side to correct it. We should set it as undefined if user doesn't set it explicitly in the `--simulation-type` option.